server/models/Card.js

```diff
@@ -10,7 +10,15 @@ const cardSchema = new mongoose.Schema({
     type: String,
     required: false,
     unique: false,
-  }
+  },
+  cardDetails: [
```

Comment:
this is an array of card details in the `cardSchema`? curious to understand why an array? would there be multiple details to a card?

Reply:
My fault, it was a mistake.